doc/terminal-v2-roadmap.md

```diff
@@ -0,0 +1,76 @@
+
+## Terminal Roadmap / Timeline
+
+| Milestone end date | Milestone Name | Preview Release Blog Post |
+| ------------------ | -------------- | ------------------------- |
+| 2020-06-30 | [1.1](https://github.com/microsoft/terminal/milestone/24) in Windows Terminal Preview |
+| 2020-07-31 | [1.2](https://github.com/microsoft/terminal/milestone/25) in Windows Terminal Preview<br>[1.1](https://github.com/microsoft/terminal/milestone/24) in Windows Terminal | |
+| 2020-08-31 | 1.3 in Windows Terminal Preview<br>[1.2](https://github.com/microsoft/terminal/milestone/25) in Windows Terminal | |
+| 2020-09-30 | 1.4 in Windows Terminal Preview<br>1.3 in Windows Terminal | |
+| 2020-10-31 | 1.5 in Windows Terminal Preview<br>1.4 in Windows Terminal | |
+| 2020-11-30 | 1.6 in Windows Terminal Preview<br>1.5 in Windows Terminal | |
+| 2020-12-31 | 1.7 in Windows Terminal Preview<br>1.6 in Windows Terminal | |
+| 2021-01-31 | 1.8 in Windows Terminal Preview<br>1.7 in Windows Terminal | |
+| 2021-02-28 | 1.9 in Windows Terminal Preview<br>1.8 in Windows Terminal | |
+| 2021-03-31 | 1.10 in Windows Terminal Preview<br>1.9 in Windows Terminal | |
+| 2021-04-30 | 2.0 RC in Windows Terminal Preview<br>2.0 RC in Windows Terminal | |
+| 2021-05-31 | [2.0](https://github.com/microsoft/terminal/milestone/22) in Windows Terminal Preview<br>[2.0](https://github.com/microsoft/terminal/milestone/22) in Windows Terminal | |
+
+## Issue Triage & Prioritization
+
+Incoming issues/asks/etc. are triaged several times a week, labeled appropriately, and assigned to a milestone in priority order:
+
+* P0 (serious crashes, data loss, etc.) issues are scheduled to be dealt with ASAP
+* P1/2 issues/features/asks  assigned to the current or future milestone, or to the [Terminal 2.0 milestone](https://github.com/microsoft/terminal/milestone/22) for future assignment, if required to deliver a 2.0 feature
+* Issues/features/asks not on our list of 2.0 features are assigned to the [Terminal Backlog](https://github.com/microsoft/terminal/milestone/7) for subsequent triage, prioritization & scheduling.
+
+## 2.0 Scenarios
+
+The following are a list of the key scenarios we're aiming to deliver for Terminal 2.0.
+
+> 👉 Note: There are many other features that don't fit within 2.0, but will be re-assessed and prioritized for 3.0, the plan for which will be published in 2021.
+
+| Priority\* | Scenario | Description/Notes |
+| ---------- | -------- | ----------------- |
+| 0 | Settings UI | A user interface that connects to settings.json. This provides a way for people to edit their settings without having to edit a JSON file.<br><br>Issue: [#1564] |

```

Comment:
I'm not sure github will auto-link issues in `[#1564]` format within a Markdown doc if you don't add the following to the bottom of the doc:
```md
[#1564]: https://github.com/microsoft/terminal/issues/1564
```

(for each and every one)

Reply:
Oh sad 😥

Reply:
![image](https://user-images.githubusercontent.com/11050425/84181641-1083f880-aa3e-11ea-8b31-16acd7b32da3.png)
Try this, maybe?
Reference: https://guides.github.com/features/mastering-markdown/

Reply:
No bueno :(